### ISSUE TITLE
Italian regions vs. provinces for Trentino-AltoAdige

### ISSUE THREAD

**Issue by ressel** [neherlab/covid19_scenarios#341]
Italy consists of 
20 regions (https://en.wikipedia.org/wiki/Regions_of_Italy) and 
103 provinces (https://en.wikipedia.org/wiki/Provinces_of_Italy).

populationData.tsv does contain data for the 20 regions.

But case data is available for only 19 of these regions.  The one exception is Trentino-South Tyrol, where the case data for its two autonomous provinces P.A. Bolzano (South Tyrol) and P.A. Trento (Trentino) is provided.  The case data for the region Trentino-South Tyrol (other name: Trentino-AltoAdige) is missing.  

Currently you cannot fairly compare and verify simulation and empirical data for any of these regions or provinces.

I propose to add 
1. case data for ITA-TrentinoAltoAdige
2. population data for ITA-South Tyrol and ITA-Trentino if possible (as these two stand out as the only two autonomous provinces).

This would allow for simulation and comparison with empirical data for this region on region- and province-level.

**Comment by rneher** [comment]
@ressel do you have population sizes and hospital data for these regions?

**Comment by ressel** [comment]
Here is what I found (two more lines for populationData.tsv):

name	populationServed	ageDistribution	hospitalBeds	ICUBeds	importsPerDay	hemisphere	srcPopulation
ITA-P.A. Bolzano	531178	Italy	1895	64	5	Northern	Wikipedia
ITA-P.A. Trento	541.098 Italy	2132	69	5	Northern	Wikipedia

Population sources (31 dec 2018):
https://de.wikipedia.org/wiki/S%C3%BCdtirol
https://de.wikipedia.org/wiki/Trentino

Hospital beds (2016)
http://www.europaregion.info/de/zahlen-europaregion.asp

ICU beds (from current numbers)
https://de.wikipedia.org/wiki/COVID-19-Pandemie_in_Italien

In addition to this, the epidemic values for ITA-TrentionAltoAdige would have to be
calculated as the sum of the values for 'ITA-P.A. Bolzano' and
'ITA-P.A. Trento'.

Hope that helps.  Happy Easter!

**Comment by noleti** [comment]
@ressel I created https://github.com/neherlab/covid19_scenarios/pull/553, can you double-check if I am doing the right thing?

**Comment by ressel** [comment]
Looking good to me.

How about extracting a method for summation of empirical data?

**Comment by noleti** [comment]
@ressel Great idea, I added this now. This also saves some of the old code in the italy parser.

**Comment by ressel** [comment]
Hi Nils,
I cannot find your changes in
https://covid19-scenarios-git-feat-italyregions.covid19-scenarios.now.sh/
(no Scenario "ITA-P.A. Bolzano", no confirmed cases "ITA-TrentinoAltoAdige").
Wrong expectations?
Cheers, Matthias

**Comment by noleti** [comment]
Thanks for checking. Things will only appear there if they are included in the generated `src/assets/data/scenarios/scenarios.json`, and I did not include that in the PR. Updating those main datastructures is done by @rneher and @nnoll , so I did not want to have them in the PR.

Unrelated to that, there was also a typo in the string when I introduced the function in utils. https://github.com/neherlab/covid19_scenarios/commit/ddfccead6cceed0fa401847620878d245b909ac5 fixed this.
